src/index.html

```diff
@@ -10,7 +10,29 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__body">
+    <form action="#"
+          class="form form-big"
+          data-qa="big"
+    >
```

Comment:
```suggestion
    <form 
      action="#"
      class="form form-big"
      data-qa="big"
    >
```
Pay attention to the formatting